tests/runner.go

```diff
@@ -0,0 +1,38 @@
+package main
```

Comment:
Do we need Go program here? Do we plan to add more complex logic? If not I'd consider simple /bin/sh script.

Reply:
Yes you're right - we no longer have any significant logic to warrant a Go program. Removed here https://github.com/aquasecurity/tracee/pull/1282/commits/6d3be669a2e7ce253508aa49560260168816f4bf